src/main/java/com/shopify/model/ShopifyVariantUpdateRequest.java

```diff
@@ -328,8 +328,8 @@ public FulfillmentServiceStep withInventoryPolicy(final InventoryPolicy inventor
 		}
 
 		@Override
-		public InventoryPolicyStep withInventoryManagement(final InventoryManagement inventoryManagement) {
-			if (shopifyVariant.getInventoryManagement() != inventoryManagement) {
+		public InventoryPolicyStep withInventoryManagement(final String inventoryManagement) {
+			if (!shopifyVariant.getInventoryManagement().equals(inventoryManagement)) {
```

Comment:
Please change this to use `doesNotEquals` method already present in this class.

Reply:
ah, completely missed that. 
Pushed my changes now.